package.json

```diff
@@ -10,7 +10,8 @@
   },
   "devDependencies": {
     "eslint": "3",
-    "mocha": "3"
+    "mocha": "3",
+    "eslint-plugin-mocha": "4.9.0"
```

Comment:
@seanmonstar ok did some more testing and it actually works both in `dependencies` and `devDependencies`.  The fxa-auth-server test failed with both of them.

So given that it works both ways, where would you like the dependency to be?